src/02-common-principles.md

```diff
@@ -363,6 +363,102 @@ In particular, if a BIDS dataset contains a `derivatives/` sub-directory,
 the contents of that directory may be a heterogeneous mix of BIDS Derivatives
 datasets and non-compliant derivatives.
 
+## BIDS URI: Pointing to files within and outside of BIDS datasets
+
+Throughout the BIDS specification there a meta data fields that contain a
+pointer to a file.
+For example `IntendedFor`, `SpatialReference`, and `AssociatedEmptyRoom`.
+Prior to BIDS version 1.6.1, such pointers were required to be "relative paths".
+However the specification failed to consistently  describe relative to *what*
+these pointers should be specified, and the exact syntax to be used.
+Yet even with perfectly well and consistently defined "relative paths", this
+method of pointing to files becomes impractical when referring to files that
+are outside of a particular file tree (for example, on the Internet), or when
+the location of the data that is being pointed to changes.
+
+For these reasons, starting with BIDS version 1.6.1, using "relative paths" for
```

Comment:
Comment by @effigies: If this PR goes in, it should be a 1.7.0 release, because this is too major a change

Reply:
I don't believe that windows file systems have a single root. I don't think the backslashes are a problem but expressing things relative to an assumed root of the file system might be.

I think doing something like this PR is absolutely necessary, but I am still uneasy about the repercussions for downstream users of having broken links to needed data when the data set is used elsewhere.  

I don't have a brilliant suggestion about how to solve this, but maybe having outside links that are not globally accessible to others should be discouraged unless absolutely necessary and should be documented somewhere as part of the data set.  Having the validator always output a warning when  the link is to a non-globally accessible resource is important --- this encourages the data provider to at least think about it.  However, there also needs to be some information provided to the downstream user.  I just don't have a good idea of how to do this.

Reply:
P.S.   Prefer always using local to describe paths relative to the data set root.

Reply:
Let me slightly amend my statement about system root.  There is a set systemroot command that allows users to set a system root to whatever they want. (My windows 10 machine seems to have it set as C:|WINDOWS by default.)  While it is theoretically possible to mount drives in empty directories on Windows,  no Windows user that I know would ever consider doing that.  There must be a cleaner way of doing this.

Reply:
@VisLab The setting you're referring to has nothing to do with the file system root we were talking about 🙂 instead, it's a pointer to the Windows system directory. 

See
https://docs.microsoft.com/en-us/windows/deployment/usmt/usmt-recognized-environment-variables

Reply:
one more windows note (I am testing right now), typing this into a browser window (firefox,chrome, edge) DOES NOT work (replace your username as fits):

`file://C/Users/stefan/Desktop`

However, this does (notice the colon after `C`):

`file://C:/Users/stefan/Desktop`

interestingly, the location then shown as: `file:///C:/Users/stefan/Desktop/` (note the additional leading `/`) on chrome and firefox, but not edge (edge just doesn't expand the `file://` scheme, see screenshot below).

![image](https://user-images.githubusercontent.com/9084751/126075906-da9b4dc5-69eb-40ff-a678-418faf42a4d5.png)

Reply:
I fixed the proposal to include the colon for Windows. Paths like `file:///C:/Users/username/Desktop/` should work on Windows (replace username with your username)